vlanBridge.go

```diff
@@ -436,3 +442,36 @@ func (vl *VlanBridge) processArp(pkt protocol.Ethernet, inPort uint32) {
 		}
 	}
 }
+
+// SendGARP sends GARP for the specified IP, MAC
+func (vl *VlanBridge) SendGARP(ip net.IP, mac net.HardwareAddr, vlanID uint16) error {
+	garpPkt, _ := protocol.NewARP(protocol.Type_Request)
```

Comment:
I wonder why we need to expose this API. This function could be self-contained...
In any case, at least we should validate the mac/IP that it exists as part of this datapath, before sending out the GARP.